pkg/kubelet/pod_actions.go

```diff
@@ -105,7 +106,8 @@ func (kl *Kubelet) DoPodAction(action *v1.Action, pod *v1.Pod) {
 	}
 	actionStatus.Error = errStr
 	action.Status = actionStatus
-	if _, err := kl.kubeClient.CoreV1().Actions(action.Namespace).UpdateStatus(action); err != nil {
+	tenantPartitionClient := kubeclientmanager.ClientManager.GetTPClient(kl.kubeTPClients, action.Tenant)
+	if _, err := tenantPartitionClient.CoreV1().Actions(action.Namespace).UpdateStatus(action); err != nil {
```

Comment:
Actions() -> ActionsWithMultiTenancy(ns, tenant)

Reply:
fixed

Reply:
OK. Please consider it resolved.